src/scripts/main.js

```diff
@@ -1 +1,87 @@
-'use strict';
+const body = document.querySelector('body');
+
+function createNotification(message, type) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = type;
+  notification.innerText = message;
+  body.insertAdjacentElement('afterbegin', notification);
+}
+
+function firstPromise() {
+  return new Promise((resolve, reject) => {
+    let isClicked = false;
+
+    const timeOut = setTimeout(() => {
+      createNotification('First promise was rejected', 'error');
```

Comment:
You should send move this function invocations to the `then` and `catch` methods.